goaviatrix/client.go

```diff
@@ -46,7 +46,7 @@ type APIRequest struct {
 //go:generate moq -rm -out client_mock.go . ClientInterface
 type ClientInterface interface {
 	DeleteAccount(account *Account) error
-	GetAccount(account *Account) (*Account, error)
```

Comment:
Whats the reason for this? Seems logical to keep pointer for consistency?

Reply:
This is the point of the change.  We return an object now instead of a pointer.  If we pass a pointer back from the cache that could cause issues, so we make a copy instead.

Reply:
Added comment to GetAccount